coccinelle/new-error.cocci

```diff
@@ -0,0 +1,19 @@
+// Conversion for G_IO_ERROR_FAILED that could be glnx_throw()
+@@
+expression p;
+@@
+- g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED, p);
+- return FALSE;
++ return glnx_throw (error, "%s", p);
```

Comment:
I tried but can't get it to work.  I don't understand why honestly.  It'd be nice, but I struggle to balance the time learning Coccinelle vs the benefit.